doc/DOCUMENTATION.org

```diff
@@ -2078,8 +2078,8 @@ Basic =ido= operations can be done with ~Ctrl~ key:
 
 | Key Binding        | Description                                       |
 |--------------------+---------------------------------------------------|
-| ~C-<return>~       | open a =dired buffer=                             |
-| ~M-<return>~       | open a =dired buffer= in terminal                 |
+| ~C-RET~            | open a =dired buffer=                             |
+| ~M-RET~            | open a =dired buffer= in terminal                 |
```

Comment:
I'm not sure this is bound anymore. Can you check and change (or remove) the description as necessary?